tests/Transport/ValueParserTest.php

```diff
@@ -28,7 +28,7 @@ public function providerParse() : iterable
         yield 'IpAddress' => ['IpAddress: 127.0.0.1', '127.0.0.1'];
         yield 'OID' => ['OID: .1.2.3', '.1.2.3'];
         yield 'STRING' => ['STRING: "abc"', 'abc'];
-        yield 'empty string' => ['""', ''];
```

Comment:
Proč? Já si tohle nevymyslel, bylo to tam protože se taková hodnota reálně objevuje

Reply:
Já pouštěl strategii a objevilo se tam to co jsem dal do testu. Myslel jsem, že je bug v testu, commity byly nějak squashnuty, tak jsem už nenašel kontext. Takže tohle je taky valid?

Reply:
Nepouštěl si to zase na Macu? To jsme řešili naposled, že se ti to chování nějak crazy ... to co tam bylo předtím je 100 % validní. Na tenhle tvůj case jsem nikde nenarazil. Kdyžtak hoď do slacku kde přesně to je a testnu to.